db/migrate/20180208042902_add_details_to_case.rb

```diff
@@ -0,0 +1,27 @@
+class AddDetailsToCase < ActiveRecord::Migration[5.1]
+  def change
+    add_column :cases, :type_of_case, :string
```

Comment:
Is this type going to be from a fixed set of possible values? If so then I'd recommend using an enum

Reply:
Yeah, that makes sense. ty ken